src/vs/workbench/parts/tasks/electron-browser/task.contribution.ts

```diff
@@ -134,7 +134,7 @@ class BuildStatusBarItem extends Themable implements IStatusbarItem {
 		let callOnDispose: IDisposable[] = [];
 
 		const element = document.createElement('div');
-		const label = document.createElement('a');
```

Comment:
@LeonardoBraga this one needs to be an `anchor` because it is clickable

Reply:
My apologies - I missed the click handler in the code right below.